domain/schema/model.go

```diff
@@ -568,11 +596,15 @@ CREATE UNIQUE INDEX idx_storage_instance_volume
 ON storage_instance_volume (storage_volume_uuid);
 
 CREATE TABLE storage_volume_attachment (
-    uuid                TEXT PRIMARY KEY,
-    storage_volume_uuid TEXT NOT NULL,
-    net_node_uuid       TEXT NOT NULL,
-    life_id             INT NOT NULL,
-    -- TODO (manadart 2023-12-11) info, params and plans.
+    uuid                   TEXT PRIMARY KEY,
+    storage_volume_uuid    TEXT NOT NULL,
+    net_node_uuid          TEXT NOT NULL,
+    life_id                INT NOT NULL,
+    device_name            TEXT,
+    device_link            TEXT,
+    bus_address            TEXT,
```

Comment:
These fields are currently only populated by maas on node creation and are used to match the model's volume with a block device polled from the machine. They are essentially a subset of what a block device has. They have no ongoing utility beyond the time when an observed block device have been associated with the volume. I wonder if we can offload these to another table like "pending_volume_attachment" or something and delete the row once we're finished with it. And we can take a step back and ensure that if/when we modernise other providers (see separate comment) it all still fits.

Reply:
Done.